pkg/registry/policy/rest/storage_policy.go

```diff
@@ -24,6 +24,7 @@ import (
 	serverstorage "k8s.io/apiserver/pkg/server/storage"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/apis/policy"
+	pspstore "k8s.io/kubernetes/pkg/registry/extensions/podsecuritypolicy/storage"
```

Comment:
Should this be extensions or policy?

Reply:
There's only a single registry and it lives under extensions for now. We'll move it as a follow up, but it's internal only